### DISCUSSION TITLE
Issue with trouble

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#3378] [General]
Just updated to latest LazyVim and started getting this
![image](https://github.com/LazyVim/LazyVim/assets/8222059/274dab58-0357-45e0-a202-99b1db55c171)

Also removed the trouble-v3 as suggested

**Comment by folke** [comment]
You did not update trouble yet (`:Lazy`)

**Reply by rubiin** [reply]
Darn . That was the issue . Had lazy check for updates every 4 hours so thats why it did not caught the update i guess
